ci/linux-packages/yum/almalinux-8/Dockerfile

```diff
@@ -40,5 +39,10 @@ RUN \
     rpmdevtools \
     sqlite-devel \
     tar \
+    wget \
     vala && \
   yum clean ${quiet} all
+
+RUN wget --no-verbose https://go.dev/dl/go1.21.8.linux-amd64.tar.gz
+RUN tar -C /usr/local -xzf go1.21.8.linux-amd64.tar.gz
```

Comment:
It's better that we do them in one `RUN` for better Docker cache.

Can I push some commits to this branch?

Reply:
absolutely, go for it

Reply:
So doing this I ended up having an issue with the packaging step because the SPEC file references the golang rpm package but we didn't install it, we're using this manual installation. 

So we'll probably have to update the SPEC file to check if go is installed, and do something if it isn't or if it's too old of a version.

Reply:
Right. :-)
I'll do it too.

Reply:
@kou Let me know if you need any assistance from me on this. There's another change waiting on us getting this up and running :smile:

Reply:
Oh, sorry. I couldn't work on this. I'll work on in a few days.

Reply:
I'd like to start the release today or tomorrow. Maybe we can include this in the next release then?

Reply:
Oh, sorry. I didn't know the schedule. I'll work on this today.